ui/store/actions.test.js

```diff
@@ -483,6 +483,51 @@ describe('Actions', () => {
     });
   });
 
+  describe('#getHardwareDeviceName', () => {
+    afterEach(() => {
+      sinon.restore();
+    });
+
+    it('calls getHardwareDeviceName in background', async () => {
+      const store = mockStore();
+
+      const getHardwareDeviceName = background.getHardwareDeviceName.callsFake(
+        (_, __, cb) => cb(),
+      );
+
+      setBackgroundConnection(background);
+
+      await store.dispatch(
+        actions.getHardwareDeviceName('ledger', "m/44'/60'/0'/0/0"),
```

Comment:
Nit: We could also have this in the scope above to be more DRY

Reply:
i have move the 'ledger' to a const variable called `deviceName`
and "m/44'/60'/0'/0/0" to `hdPath` to below describe.